assets/overlays/openstack-manila/patches/controller_add_driver.yaml

```diff
@@ -33,16 +26,6 @@ spec:
         - key: node-role.kubernetes.io/master
           operator: Exists
           effect: "NoSchedule"
-      affinity:
```

Comment:
Indeed, this is normally added by the [/assets/common/standalone/controller_add_affinity.yaml](https://github.com/openshift/csi-operator/blob/master/assets/common/standalone/controller_add_affinity.yaml) file.